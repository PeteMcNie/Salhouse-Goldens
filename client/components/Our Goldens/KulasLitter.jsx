import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'
import Image from 'react-bootstrap/Image'

function KulasLitter () {
  return (
    <><div className='mainBody'>
      <Container>
        <Row className='row'>
          <Col>
            <h2 className='litterTitle'>Kula&apos;s Litter 2006</h2>
          </Col>
        </Row>
        <Row>
          <Col md={6} className='leftCol'>
            <Image src="images/kl1.jpg" fluid className='litterMainImage'/>
            <p className='litterMainText'>Kula</p>
          </Col>
          <Col md={6}>
            <Image src="images/kl2.jpg" fluid className='litterMainImage'/>
            <p className='litterMainText'>Jordan <a href='https://www.mistymoorgoldens.com/'>(NZ CH Santamaria The Patriot)</a></p>
          </Col>
        </Row>

        <hr className='sectionSeparator' />

        <Row>
          <Col lg={6} className='leftCol'>
            <Image src="images/kl3.jpg" fluid className='litterImage'/>
            <p className='litterText'>Kula 8 weeks pregnant (like a beached whale!).</p>
          </Col>
          <Col lg={6}>
            <Image src="images/kl4.jpg" fluid className='litterImage'/>
            <p className='litterText'>Our first ever puppy born!</p>
          </Col>
        </Row>
        <Row>
          <Col lg={6} className='leftCol'>
            <Image src="images/kl5.jpg" fluid className='litterImage'/>
            <p className='litterText'>We had a lot of fun with these puppies and of course kept the lovely Shadow.</p>
          </Col>
          <Col lg={6}>
            <Image src="images/kl6.jpg" fluid className='litterImage'/>
            <p className='litterText'>Kula wonders what is moving under the blanket!</p>
          </Col>
        </Row>
        <Row>
          <Col lg={6} className='leftCol'>
            <Image src="images/kl7.jpg" fluid className='litterImage'/>
            <p className='litterText'>Pileup of puppies!</p>
          </Col>
          <Col lg={6}>
            <Image src="images/kl8.jpg" fluid className='litterImage'/>
            <p className='litterText'>Puppies feeding.</p>
          </Col>
        </Row>
        <Row>
          <Col lg={6} className='leftCol'>
            <Image src="images/kl9.jpg" fluid className='litterImage'/>
            <p className='litterText'>Asleep under the barbecue.</p>
          </Col>
          <Col lg={6}>
            <Image src="images/kl10.jpg" fluid className='litterImage'/>
            <p className='litterText'>All on Jeff!</p>
          </Col>
        </Row>
        <Row>
          <Col lg={6} className='leftCol'>
            <Image src="images/kl11.jpg" fluid className='litterImage'/>
            <p className='litterText'>Let&apos;s try out the pool.</p>
          </Col>
          <Col lg={6}>
            <Image src="images/kl12.jpg" fluid className='litterImage'/>
            <p className='litterText'>1st birthday - all except the second female here.</p>
          </Col>
        </Row>
      </Container>
    </div>
    </>
  )
}

export default KulasLitter
