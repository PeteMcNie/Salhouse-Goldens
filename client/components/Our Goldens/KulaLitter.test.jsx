import React from 'react'
import { render } from '@testing-library/react'

import KulasLitter from './KulasLitter'

test('KulasLitter component matches snapshot', () => {
  const { container } = render(<KulasLitter />)
  expect(container).toMatchSnapshot()
})
