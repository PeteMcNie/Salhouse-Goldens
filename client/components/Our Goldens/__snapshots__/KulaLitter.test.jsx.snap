// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KulasLitter component matches snapshot 1`] = `
<div>
  <div
    class="mainBody"
  >
    <div
      class="container"
    >
      <div
        class="row row"
      >
        <div
          class="col"
        >
          <h2
            class="litterTitle"
          >
            Kula's Litter 2006
          </h2>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-md-6"
        >
          <img
            class="litterMainImage img-fluid"
            src="images/kl1.jpg"
          />
          <p
            class="litterMainText"
          >
            Kula
          </p>
        </div>
        <div
          class="col-md-6"
        >
          <img
            class="litterMainImage img-fluid"
            src="images/kl2.jpg"
          />
          <p
            class="litterMainText"
          >
            Jordan 
            <a
              href="https://www.mistymoorgoldens.com/"
            >
              (NZ CH Santamaria The Patriot)
            </a>
          </p>
        </div>
      </div>
      <hr
        class="sectionSeparator"
      />
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl3.jpg"
          />
          <p
            class="litterText"
          >
            Kula 8 weeks pregnant (like a beached whale!).
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl4.jpg"
          />
          <p
            class="litterText"
          >
            Our first ever puppy born!
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl5.jpg"
          />
          <p
            class="litterText"
          >
            We had a lot of fun with these puppies and of course kept the lovely Shadow.
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl6.jpg"
          />
          <p
            class="litterText"
          >
            Kula wonders what is moving under the blanket!
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl7.jpg"
          />
          <p
            class="litterText"
          >
            Pileup of puppies!
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl8.jpg"
          />
          <p
            class="litterText"
          >
            Puppies feeding.
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl9.jpg"
          />
          <p
            class="litterText"
          >
            Asleep under the barbecue.
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl10.jpg"
          />
          <p
            class="litterText"
          >
            All on Jeff!
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl11.jpg"
          />
          <p
            class="litterText"
          >
            Let's try out the pool.
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/kl12.jpg"
          />
          <p
            class="litterText"
          >
            1st birthday - all except the second female here.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
